src/importer.js

```diff
@@ -0,0 +1,50 @@
+const fs = require('fs');
+const EventEmitter = require('events').EventEmitter;
+const csvjson = require('csvjson');
+
+const checkedFiles = [];
+
+class Importer extends EventEmitter {
+  constructor(dirWatcher, path) {
+    super();
+    // sync call:
+    // dirWatcher.on('data change', () =>
+    //   this.importSync(path)
+    // );
+    // ---------------
+    // async call:
+    dirWatcher.on('data change', () =>
```

Comment:
it was probably missed in the task description, but it's better to move subscribing from `constructor` to some external method, like `listen(fsWatcher)`. You also don't need to pass path there, because your watcher already use some path to watch, and you should use same path here. running business logic from constructor is also side effect and sign of bad design, just imagine if someone will need to import files only once, without watching them - this class wont fit, because it run watching logic implicitly. so it's much better to make it explicit and move from constructor to separate method, i dunno why they wasn't mention this in task description